roles/cloud_connector/templates/foreman_rh_cloud.toml.j2

```diff
@@ -0,0 +1,7 @@
+exec = “/usr/libexec/yggdrasil-worker-forwarder”
```

Comment:
Maybe this is just me, but the quotes look like U+201D `“` instead of U+0022 `"`

Reply:
Goood catch.

Reply:
Nice catch! The suggestion was copied from google doc that autocompleted the quotes :facepalm: